utils/validation.js

```diff
@@ -0,0 +1,71 @@
+let users = [
+      {
+            traineeEmail: "prnv@successive.tech",
+            reviewerEmail: "pandey@successive.tech"
+      },
+      {
+            traineeEmail: "prnv@successive.tech*1",
+            reviewerEmail: "pandey@successive.tech"
+      },
+      {
+            traineeEmail: "prnv@successive.tech",
+            reviewerEmail: "pandey@successive.tech"
+      },
+      {
+            traineeEmail: "prnv@successive.tech",
+            reviewerEmail: "pandey@successive.tech"
+      },
+      {
+            traineeEmail: "prnv@successive.te/",
+            reviewerEmail: "pandey@successive.tech//*"
+      },
+      {
+            traineeEmail: "prnv@successive.te/*",
+            reviewerEmail: "pandey@successive.t"
+      },
+      {
+            traineeEmail: "prnv@successive.tech",
+            reviewerEmail: "pandey@successive.tech"
+      },
+      {
+            traineeEmail: "prnv@successive.tech98",
+            reviewerEmail: "pandey@successive.tech"
+      }
+];
+
+//Will check her formating of emails
+let checkemail = /^[a-zA-Z0-9._-]+@[successive]+\.[tech]{2,4}$/;
+
+let valid_users=[];
+let invalid_users=[];
+
+// function for validate the email.
+function validateEmail(email){
+      return checkemail.test(String(email).toLowerCase());
+}
+
+//function for validate the users
+function validateUsers(users)
+{   
+    users.forEach((users)=>{
+    const {traineeEmail,reviewerEmail} = users;
+    if(validateEmail(traineeEmail) && validateEmail(reviewerEmail)){
```

Comment:
change variable name of users in this forEach((users). This is already define in the uper scope